docs/reference/Image.rst

```diff
@@ -363,6 +363,7 @@ Classes
     :show-inheritance:
 .. autoclass:: PIL.Image.ImagePointHandler
 .. autoclass:: PIL.Image.ImageTransformHandler
+.. autoclass:: PIL.Image._E
```

Comment:
Do we want this "empty" internal class in the docs?

<img width="764" alt="image" src="https://github.com/user-attachments/assets/978e4570-7fa5-46a4-b1ce-88838e5656b3">

https://pillow--8319.org.readthedocs.build/en/8319/reference/Image.html#PIL.Image._E

Reply:
It's referenced from the type hint of https://pillow--8319.org.readthedocs.build/en/8319/reference/Image.html#PIL.Image.Image.point

Reply:
For a user-facing class, can we give it a description or a better name?

Reply:
I've added a description, and deprecated it in favour of a new name, `ImagePointTransform`.